### PR TITLE
Updating allow list requirements

### PR DESCRIPTION
**Pull request by somakbhattacharyya**

### CODE REVIEW

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/sign-in-teams.md [hunk 1]
```diff
@@ -53,6 +53,9 @@ If users are working on a domain-joined computer (for example, if their tenant h
 ## Signing out of Microsoft Teams after completing modern authentication
 To sign out of Teams, users can click their profile picture at the top of the app, and then select **Sign out**. They can also right-click the app icon in their taskbar, and then select **Log out**. Once they've sign out of Teams, they need to enter their credentials again to launch the app.
 
+## URLs and IP address ranges
+Teams requires connectivity to the Internet. To understand endpoints that should be reachable for customers using Teams in Office 365 plans, Government and other clouds, please read the [guidance available here](https://docs.microsoft.com/en-us/office365/enterprise/urls-and-ip-address-ranges). In addition to this, you'd need to also allow access to https://ssl.gstatic.com.
```
Comment (JohanFreelancer9) [inline Teams/sign-in-teams.md]:
```suggestion
Teams requires connectivity to the Internet. To understand endpoints that should be reachable for customers using Teams in Office 365 plans, Government and other clouds, please read [Office 365 URLs and IP address ranges](https://docs.microsoft.com/office365/enterprise/urls-and-ip-address-ranges). In addition to this, you also need to allow access to https://ssl.gstatic.com.
```

**Comment by scanum** [comment]
@LolaJacobsen PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by LolaJacobsen** [comment]
Changes per PM @somakbhattacharyya

**Pull request merged**